packages/presentation/src/components/FilePreview.svelte

```diff
@@ -34,9 +34,12 @@
   let minHeight: number | undefined
   $: parentHeight = ($deviceInfo.docHeight * 80) / 100
 
+  let loading = true
   let previewType: FilePreviewExtension | undefined = undefined
   $: void getPreviewType(contentType, $previewTypes).then((res) => {
+    loading = true
```

Comment:
Minor: sync block setting loading to true and then to false makes no sense?

Reply:
😂